compiler/rustc_const_eval/src/interpret/step.rs

```diff
@@ -303,6 +303,21 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                 let variant_dest = self.project_downcast(dest, variant_index)?;
                 (variant_index, variant_dest, active_field_index)
             }
+            mir::AggregateKind::RawPtr(..) => {
+                // Trying to `project_field` into pointers tends not to work,
```

Comment:
What does "tends to" mean?^^

But indeed constructing a new Immediate is the intended way to do this.

Reply:
Yeah -- only wide pointers have "fields" (and `RawPtr` also works for thin pointers it seems), and those fields have weird types which codegen doesn't care about much but Miri does.

Reply:
Thanks! interpreter changes look good to me.